testdata/ede_caching_subnetcache.tdir/ede.conf

```diff
@@ -0,0 +1,25 @@
+server:
+	verbosity: 3
+	interface: 127.0.0.1
+	port: @PORT@
+	use-syslog: no
+	directory: .
+	pidfile: "unbound.pid"
+	chroot: ""
+	username: ""
+	directory: ""
+	val-log-level: 2
+
+	auto-trust-anchor-file: "root.key"
```

Comment:
Is the trust anchor needed?

Reply:
These tests no longer exist; mark as resolved.